### ISSUE TITLE
`mapObjIndex` cannot convert undefined or null to object

### ISSUE THREAD

**Issue by tuckergordon** [selfrefactor/rambda#641]
The following works in ramda: ([REPL](https://ramdajs.com/repl/?v=0.28.0#?%2F%2F%20INPUT%0Aconst%20myObj%20%3D%20%7B%0A%20%20foo%3A%20%7B%0A%20%20%20%20bar%3A%20%7B%0A%20%20%20%20%20%20a%3A%20%7B%0A%20%20%20%20%20%20%20%20myProp%3A%20%27some%20value%27%0A%20%20%20%20%20%20%7D%2C%0A%20%20%20%20%20%20b%3A%20%7B%0A%20%20%20%20%20%20%20%20myProp%3A%20%27some%20other%20value%27%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%3B%0A%0A%2F%2F%20RAMDA%0A%0Aconst%20result%20%3D%20R.pipe%28%0A%20%20R.path%28%5B%27foo%27%2C%20%27bar%27%5D%29%2C%0A%20%20R.defaultTo%28%7B%7D%29%2C%0A%20%20R.mapObjIndexed%28%28val%2C%20key%29%20%3D%3E%20%28%7B%20key%2C%20...val%20%7D%29%29%2C%0A%20%20R.values%0A%29%3B%0A%0A%0A%2F%2F%20PRINTING%0A%0Aconst%20prettyPrint%20%3D%20%28obj%29%20%3D%3E%20console.log%28JSON.stringify%28obj%2C%20null%2C%20%27%20%20%27%29%29%3B%0A%0Aconsole.log%28%27myObj%3A%27%29%0AprettyPrint%28myObj%29%3B%0A%0Aconsole.log%28%27Result%3A%27%29%0AprettyPrint%28result%28myObj%29%29%3B))

```js
// INPUT
const myObj = {
  foo: {
    bar: {
      a: {
        myProp: 'some value'
      },
      b: {
        myProp: 'some other value'
      }
    }
  }
};

// RAMDA

const result = R.pipe(
  R.path(['foo', 'bar']),
  R.defaultTo({}),
  R.mapObjIndexed((val, key) => ({ key, ...val })),
  R.values
);
```
Result:
```js
[
  {
    "key": "a",
    "myProp": "some value"
  },
  {
    "key": "b",
    "myProp": "some other value"
  }
]
```

However, the same code in the rambda REPL results in an error coming from `R.mapObjIndexed`
```zsh
Cannot convert undefined or null to object
    at keys (<anonymous>)
```

Still digging into this more to try to pinpoint exactly what is causing the problem, but figured it was worth reporting in the meantime

**Comment by tuckergordon** [comment]
Ah sorry should've clarified that it's happening with the library itself too - just figured the REPL would be easiest for replicating. Appreciate you looking into this!

**Comment by selfrefactor** [comment]
The function was missing currying logic. A fix is applied and it will be released with `7.2.0`. Again, thank you for the report.

**Comment by selfrefactor** [comment]
Closing the issue as fix was released with `7.2.0`
